lasagne/layers/recurrent.py

```diff
@@ -379,8 +404,8 @@ class RecurrentLayer(CustomRecurrentLayer):
            arXiv preprint arXiv:1308.0850 (2013).
     """
     def __init__(self, incoming, num_units,
-                 W_in_to_hid=init.Uniform(),
-                 W_hid_to_hid=init.Uniform(),
+                 W_in_to_hid=init.Normal(0.1),
```

Comment:
Can we address this in a different issue/PR?  I think being consistent is good, but we are also not being consistent with the rest of `lasagne` if we do this.